purchase_work_acceptance/views/purchase_views.xml

```diff
@@ -6,14 +6,15 @@
         <field name="inherit_id" ref="purchase.purchase_order_form" />
         <field name="arch" type="xml">
             <xpath expr="//button[@name='button_approve']" position="after">
+                <field name="wa_accepted" invisible="1" />
```

Comment:
Are you having this field twice in the same form?

Reply:
ah, sorry let me fix

Reply:
fixed